build/Targets/Settings.targets

```diff
@@ -201,13 +201,19 @@
   <UsingTask TaskName="Microsoft.CodeAnalysis.BuildTasks.Vbc"
              AssemblyFile="$(BootstrapBuildPath)\Microsoft.Build.Tasks.CodeAnalysis.dll"
              Condition="'$(BootstrapBuildPath)' != ''" />
+  <UsingTask TaskName="Microsoft.CodeAnalysis.BuildTasks.CheckBootstrapTask"
```

Comment:
This creates an implicit requirement that the bootstrap compiler is debug-only, right?

I've used the bootstrap logic a couple of times to test out different release versions of the compiler (for perf testing, mostly). Is there some way we could condition this aside from on the bootstrap logic?